test/test_bug3473.m

```diff
@@ -1,10 +1,11 @@
 function test_bug3473
 
-% WALLTIME 00:10:00
+% WALLTIME 00:30:00
 % MEM 2gb
 
-% DEPENDENCY ft_prepare_mesh 
+% DEPENDENCY ft_prepare_mesh
 
+addpath(genpath(dccnpath('/home/common/matlab/fieldtrip/data/test/bug3473/cpd')));
```

Comment:
should this not be handled by ft_hastoolbox?

Reply:
As we use external software for this, I add it manually

Reply:
ft_hastoolbox is meant for handling external software. It would be enough if it would just give an error and point people to the URL to download the software.

Reply:
Ok this is what it currently is doing!